### PR TITLE
WPCOM Misc Sync

### PR DESCRIPTION
**Pull request by georgestephanis**
Just a sync of some random files from WPCOM that weren't much out of sync.  Minor tidying.

### CODE REVIEW

**Comment by georgestephanis** [comment]
Friendly reminder: DO NOT SQUASH THIS PR WHEN MERGING.

Thank you!

File: modules/shortcodes/soundcloud.php [hunk 1]
```diff
@@ -57,22 +59,25 @@ function soundcloud_shortcode( $atts, $content = null ) {
...
+			'iframe' => true, // A8C: See above comment; flash is not a supported option
 			'width'  => soundcloud_get_option( 'player_width' ),
 			'height' => soundcloud_url_has_tracklist( $shortcode_options['url'] ) ? soundcloud_get_option( 'player_height_multi' ) : soundcloud_get_option( 'player_height' ),
 			'params' => array_filter(
 				array(
 					'auto_play'     => soundcloud_get_option( 'auto_play' ),
 					'show_comments' => soundcloud_get_option( 'show_comments' ),
 					'color'         => soundcloud_get_option( 'color' ),
-					'visual'        => ( $isVisual ? 'true' : 'false' ),
+					'visual'        => ( $shortcode_options['visual'] ? 'true' : 'false' ), // A8C: See above comment; always test visual from parameters
```
Comment (georgestephanis) [inline modules/shortcodes/soundcloud.php]: This is trying to rely on on a parameter that may not be set.

Comment (georgestephanis) [inline modules/shortcodes/soundcloud.php]: Fixed by @aduth in ae3a795

**Comment by dereksmart** [comment]
This looks fine to me

File: modules/theme-tools/responsive-videos/responsive-videos.js [hunk 2]
```diff
@@ -1,98 +1,67 @@
...
-	$( window ).load( responsive_videos ).resize( debounce( responsive_videos, 100 ) ).trigger( 'resize' );
-	$( document ).on( 'post-load', responsive_videos );
+	$( document ).ready( function() {
+		$( window )
+			.on( 'load.jetpack', responsiveVideos )
+			.on( 'resize.jetpack', function() {
+				clearTimeout( resizeTimer );
+				resizeTimer = setTimeout( responsiveVideos, 500 );
+			} )
+			.on( 'post-load.jetpack', responsiveVideos )
```
Comment (eliorivero) [inline modules/theme-tools/responsive-videos/responsive-videos.js]: Kudos for adding the namespace 👍

**Comment by georgestephanis** [comment]
As this has two reviews, I'll merge it myself.

**Pull request merged**